manifests/v1beta1/installs/katib-external-db/katib-config-patch.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
```

Comment:
As we discussed on the meeting, we should move this file to `/manifests/v1beta1/components/controller` and remove current `katib-config.yaml` file.

Reply:
I've removed the `katib-config-patch.yaml` files in the installation directory and move the image tags to `/manifests/v1beta1/components/controller/katib-config.yaml`.